pom.xml

```diff
@@ -123,7 +123,9 @@
     <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
     <maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
     <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
-    <maven-shade-plugin.version>3.2.3</maven-shade-plugin.version>
+    <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+    <!-- 3.0.0-M5 breaks running brave-tests ITRemoteTest.tracer_includesClassName:26
```

Comment:
to conserve energy, I'm just going to wait until M6 and see if someone else in the maven ecosystem sorts it out :P